tests/gen/README.md

```diff
@@ -0,0 +1,156 @@
+# Automatic Documents Generation 
+
+`AutoGenerate` and `AutoGenerateFromSDL` are a highly versatile function designed for dynamic document generation, perfect for testing and simulation purposes. 
+
+`AutoGenerateFromSDL` creates documents based on a specified schema, allowing for extensive customization of data generation. 
```

Comment:
todo: I think `documents based on a specified schema,` is incorrect, and should instead read something like `documents based on a specified GQL SDL, which may contain multiple schema/collection definitions,`.